src/ngx_http_lua_common.h

```diff
@@ -140,6 +140,8 @@ struct ngx_http_lua_main_conf_s {
 
     ngx_array_t         *shm_zones;  /* of ngx_shm_zone_t* */
 
+    ngx_array_t         *shdict_shm_zones; /* shm zones of "shdict" */
+
```

Comment:
I think It's a better way to store zones of "shdict", based on the current implementation.